Gemfile.lock

```diff
@@ -1,77 +0,0 @@
-GIT
-  remote: https://github.com/18F/omniauth-myusa.git
-  revision: abff3f44871a377a7000ce458898db17553d9b79
-  specs:
-    omniauth-myusa (0.0.4)
-      omniauth (~> 1.0)
-      omniauth-oauth2 (= 1.3.1)
-
-GEM
-  remote: https://rubygems.org/
-  specs:
-    aws-sdk (2.2.15)
-      aws-sdk-resources (= 2.2.15)
-    aws-sdk-core (2.2.15)
-      jmespath (~> 1.0)
-    aws-sdk-resources (2.2.15)
-      aws-sdk-core (= 2.2.15)
-    backports (3.6.8)
-    encrypted_cookie (0.0.4)
-    faraday (0.9.2)
-      multipart-post (>= 1.2, < 3)
-    hashie (3.4.3)
-    jmespath (1.1.3)
-    jwt (1.5.1)
-    kgio (2.10.0)
-    multi_json (1.11.2)
-    multi_xml (0.5.5)
-    multipart-post (2.0.0)
-    oauth2 (1.1.0)
-      faraday (>= 0.8, < 0.10)
-      jwt (~> 1.0, < 1.5.2)
-      multi_json (~> 1.3)
-      multi_xml (~> 0.5)
-      rack (>= 1.2, < 3)
-    omniauth (1.3.1)
-      hashie (>= 1.2, < 4)
-      rack (>= 1.0, < 3)
-    omniauth-oauth2 (1.3.1)
-      oauth2 (~> 1.0)
-      omniauth (~> 1.2)
-    rack (1.6.4)
-    rack-protection (1.5.3)
-      rack
-    rack-test (0.6.3)
-      rack (>= 1.0)
-    raindrops (0.15.0)
-    sinatra (1.4.7)
-      rack (~> 1.5)
-      rack-protection (~> 1.4)
-      tilt (>= 1.3, < 3)
-    sinatra-contrib (1.4.6)
-      backports (>= 2.0)
-      multi_json
-      rack-protection
-      rack-test
-      sinatra (~> 1.4.0)
-      tilt (>= 1.3, < 3)
-    tilt (2.0.2)
-    unicorn (5.0.1)
-      kgio (~> 2.6)
-      rack
-      raindrops (~> 0.7)
-
-PLATFORMS
-  ruby
-
-DEPENDENCIES
-  aws-sdk
-  encrypted_cookie
-  omniauth
-  omniauth-myusa!
-  sinatra
-  sinatra-contrib
-  unicorn
-
-BUNDLED WITH
-   1.10.6
```

Comment:
Re-added. I'm not sure _why_ I did this. I was debugging something, but didn't put a coherent reason in my commit message. :/